### PR TITLE
COA Documentation Update

### PR DESCRIPTION
**Pull request by tkainz**
Updated Banno change of address  README, JSON contract & questionnaire files

### CODE REVIEW

File: change-address/README.md [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
+A successful address update by the program or program errors encountered while attempting to update the member's address will  result in a note record added to the member's account:
 
-**
 
-## Detailed information coming soon
+## PowerOn installation
+An Overview
+* Install the programs (BANNO.CHANGE.ADDR.V1.POW & BANNO.DATABASE.CHECK.V1.POW) to the REPWRITERSPECS directory*
+* Install the configuration Letter file (BANNO.CHANGE.ADDR.V1.CFG) to the LETTERSPECS directory and update the settings as necessary*
+* Add the program to SymXchange Common parameters and refresh SymXchange
```
Comment (jucarson) [inline change-address/README.md:33]: Should this say both programs?

**Review by jucarson — CHANGES_REQUESTED** [review]
Just the 1 item!

File: change-address/README.md (hunk 1 above)
Comment (tkainz) [reply change-address/README.md:33]: Yes indeed.  TY

**Review by jucarson — APPROVED** [review]
Looks Good

**Review by jordan-keenan — APPROVED** [review]
Code looks good!

**Review by christinasimms — APPROVED** [review]
Good to go

**Pull request merged**